mulog-kinesis/src/com/brunobonacci/mulog/publishers/aws_utils.clj

```diff
@@ -0,0 +1,15 @@
+(ns com.brunobonacci.mulog.publishers.aws-utils
+  (:require
+    [cognitect.aws.client.api :as aws]))
+
+(def kinesis (aws/client {:api  :kinesis}))
```

Comment:
This is defined globally (and evaluated a compile-time) it should be moved at runtime
and use the publisher configuration to specify optional parameters (like credentials, endpoints etc

Reply:
agree, fixed